code/game/objects/structures/fake_machines/vendor.dm

```diff
@@ -237,6 +244,14 @@
 	popup.set_content(contents)
 	popup.open()
 
+/obj/structure/fake_machine/vendor/process()
+    if(obj_broken)
+        return
+    if(world.time > next_hawk)
+        next_hawk = world.time + rand(1 MINUTES, 2 MINUTES)
+        if(held_items.len)
+            say("[pick(held_items)] for sale.")
```

Comment:
It would be more cool if it shouted an item and its price and shouting will be more attention grabbing